persistence/src/main/java/org/mskcc/cmo/metadb/persistence/NormalSampleRepository.java

```diff
@@ -0,0 +1,16 @@
+package org.mskcc.cmo.metadb.persistence;
+
+import java.util.UUID;
+import org.mskcc.cmo.metadb.model.NormalSampleEntity;
+import org.springframework.data.neo4j.repository.Neo4jRepository;
+import org.springframework.stereotype.Repository;
+
+/**
+ *
+ * @author ochoaa
+ */
+
+@Repository
+public interface NormalSampleRepository extends Neo4jRepository<NormalSampleEntity, UUID> {
+    
```

Comment:
Will you be adding the custom queries to get normals for a patient/tumor sample in this PR or would this be handled by the built-in neo4j methods? I think custom queries should probably be included in this PR.

Reply:
I'm not adding any custom queries for this PR, but we might have to add them for the next card (#8)